inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -770,4 +782,13 @@ public function cancel_queues() {
 	public function delete_as_tables_transient_on_tools_page() {
 		delete_transient( 'rocket_rucss_as_tables_count' );
 	}
+
+	/**
+	 * Disable RUCSS on wrong license.
+	 *
+	 * @return bool
+	 */
+	public function disable_russ_on_wrong_license() {
+		return (bool) get_transient( 'wp_rocket_no_licence' );
+	}
```

Comment:
this method is not used?

Reply:
It is used on line 115 with the  `pre_get_rocket_option_remove_unused_css` filter